src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -786,10 +787,15 @@ public static ICC_Profile getInstance(byte[] data) {
         ProfileDataVerifier.verify(data);
         Profile p;
         try {
+            byte[] theHeader = new byte[HEADER_SIZE];
+            System.arraycopy(data, 0, theHeader, 0, HEADER_SIZE);
```

Comment:
We won't run into ArrayIndexOutOfBoundsException here since the incoming data array size is already being verified in ProfileDataVerifier.verify(data).